sdk/src/main/java/io/customer/sdk/CustomerIO.kt

```diff
@@ -337,6 +344,14 @@ class CustomerIO internal constructor(
             val client = CustomerIO(diGraph)
             val logger = diGraph.logger
 
+            // cleanup of old reference if it exists
```

Comment:
I appreciate the comment here. It is missing explanation for why we need to cleanup the old reference. It would be great to have this comment modified for that.